src/createServer.js

```diff
@@ -1,8 +1,46 @@
 'use strict';
 
+const http = require('http');
+const path = require('path');
+const fs = require('fs');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((req, res) => {
+    const pathname = req.url;
+    let normalizedPath = pathname.replace('/file', '');
```

Comment:
The path normalization logic might not handle cases where the URL contains query parameters or hash fragments. Consider using 'url.parse' to extract the pathname properly.